test/e2e/suite/certificates/othernamesan.go

```diff
@@ -141,7 +141,7 @@ YH0ROM05IRf2nOI6KInaiz4POk6JvdTb
 		Expect(cert.Extensions).To(HaveSameSANsAs(expectedSanExtension))
 	})
 
-	It("Should error if a certificate is supplied with an othername containing an invalid oid value", func() {
+	It("Should error if a certificate is supplied with another name containing an invalid oid value", func() {
```

Comment:
and this turned out to be an interesting bug...

Reply:
```suggestion
	It("Should error if a certificate is supplied with an `otherName` containing an invalid oid value", func() {
```

Maybe this way it is clearer that we are actually talking about a specific value "otherName".